frontend/css/brainzplayer.less

```diff
@@ -351,6 +351,23 @@
   }
 }
 
+.volume {
+  position: fixed;
+  width: fit-content;
+  z-index: 109;
+  height: 10em;
+  background-color: #f8f8f8;
+  right: 16em;
+  .volume-input {
+    width: 7em;
+    margin-top: @brainzplayer-height;
+    transform: rotate(270deg);
+  }
+  &.show {
+    bottom: @brainzplayer-height;
+  }
+}
+
```

Comment:
Let's make some styles common to .volume and .queue.

These are the properties I can tell will help have a consistent look between the two:
```css
  transition: bottom 1s ease-in-out;
  background-color: #f8f8f8;
  border-top: 1px solid #ccc;
  box-shadow: -6px -1px 10px rgba(0, 0, 0, 0.2);
  &.show {
    bottom: @brainzplayer-height;
  }
```